src/components/IstioWizards/WorkloadWizardDropdown.tsx

```diff
@@ -153,7 +174,17 @@ class WorkloadWizardDropdown extends React.Component<Props, State> {
       />
     );
     // TODO WorkloadWizard component contains only 3scale actions but in the future we may need to bring it back
-    return <>{dropdown}</>;
+    return (
+      <>
+        {dropdown}
+        <EnvoyDetailsModal
+          namespace={this.props.namespace}
+          workload={this.props.workload}
+          show={this.state.showWizard}
```

Comment:
Not a blocker, because today we have only "one" modal in the Workload Details.
But in previous Kiali version we used to have other modals for other extensions operations.
So, as the WorkloadWizardDropdown manages the state about when to show or not the modal, it's good to have a wrapper "WorkloadWizard" to manage this in a single place.

Otherwise when we brought back more modals in this section (others are on the way) we'll need to adjust this.

So basically the idea open the WorkloadWizard and inside the workload wizard modal says "show Envoy".

That's how it works in the Service Details.

Not a blocker, I suggest I'll send a followup PR with that, it wouldn't change anything relevant just a couple of refactoring about how to structure components.